cmake/onnxruntime_hosting.cmake

```diff
@@ -27,13 +27,43 @@ find_package(Boost 1.69 COMPONENTS system context thread program_options REQUIRE
 set(re2_src ${REPO_ROOT}/cmake/external/re2)
 
 # Setup source code
+file(GLOB_RECURSE onnxruntime_hosting_lib_srcs
+        "${ONNXRUNTIME_ROOT}/hosting/util.cc"
```

Comment:
If we are going to make a separate lib, should we move it to it's own folder?

Reply:
I will merge it with http utilities in the next PR.